src/Symfony/Component/Mime/Tests/Part/TextPartTest.php

```diff
@@ -55,6 +55,16 @@ public function testConstructorWithFile()
         $this->assertSame('content', implode('', iterator_to_array($p->bodyToIterable())));
     }
 
+    public function testConstructorWithUnknownFile()
+    {
+        $p = new TextPart(new File(\dirname(__DIR__).'/Fixtures/unknown.txt'));
+
+        // Exception should be thrown only when the body is accessed
+        $this->expectException(InvalidArgumentException::class);
```

Comment:
the class is missing a `use` statement for this exception

Reply:
oops